test/Streamly/Test/Data/Parser.hs

```diff
@@ -1129,6 +1132,29 @@ takeStartBy_ =
             where
                 predicate = odd
                 parser = P.takeStartBy_ predicate FL.toList
+
+-- Scientific notation
+scientificExpFP :: Property
+scientificExpFP =  forAll (chooseDouble (-99.99e-12, 1234.4567e+234)) $ \ls ->
```

Comment:
Can you explain why you chose this range?

Reply:
cover the value range of negative and positive with a negative and positive exponent.